src/commands/sbt_cached.yml

```diff
@@ -57,6 +61,12 @@ steps:
             name: SBT command - << parameters.cmd >>
             command: << parameters.cmd >>
 
+  - when:
+      condition: << parameters.store_test_results >>
+      steps:
+        - store_test_results:
+            path: ~/workdir
```

Comment:
do we need to store the entire `workdir`?

Reply:
It doesn't store the whole workdir. 
As it's described in the [documentation](https://circleci.com/docs/2.0/collect-test-data/) and verified in this sample [build](https://app.circleci.com/pipelines/bitbucket/qamine/remote-provider-service/1358/workflows/0bbfdea6-fcea-4df4-8b53-5453563ea713/jobs/8098) it searches all subdirectories of the specified path to look for XML test result files.